src/main/resources/validationrules/civil/civil-judicial-org-role-mapping.drl

```diff
@@ -446,6 +446,7 @@ end;
 rule "civil_task_supervisor_org_role"
 when
   $f:  FeatureFlag(status && flagName == FeatureFlagEnum.CIVIL_WA_1_0.getValue())
+  $f2: FeatureFlag(status == false, flagName == FeatureFlagEnum.CIVIL_WA_2_1.getValue())
```

Comment:
Can you add the same `Made obsolete in COT-905...` comment as used on the other rules that are being repaced.

Reply:
Resolved in e88c8c124ed76e7acca7f917670e2097b70ecabe.